softvis3d-frontend/src/classes/Layout.ts

```diff
@@ -10,6 +10,10 @@ export default class Layout implements SelectOptionValue {
         this.description = description;
     }
 
+    public getValue(): string {
```

Comment:
Looking at all 4 value classes here (Layout, Metric, Profile and Scale), some have an id as unique identifier and some have a key. I think it would be worth it to use id or key in all of the classes.